### ISSUE TITLE
Promotion video missing subtitles

### ISSUE THREAD

**Issue by normanbrobinson** [bmcmahen/Subtitles#9]
The actual video used to promote this project on http://subtitles.fiddleware.com/ hasn't been captioned and needs captioning. Source hosted on vimeo https://vimeo.com/53719196 account.

**Comment by bmcmahen** [comment]
Good point. At the time Vimeo didn't support captions, but it looks like that's changed. Thanks for the reminder to do this :)
